examples2/src/erc20.rs

```diff
@@ -125,7 +126,7 @@ impl Erc20 {
     pub fn cross_transfer(&mut self, other: Address, to: Address, value: U256) {
         let caller = self.env().caller();
 
-        let other_erc20 = Erc20ContractRef {
+        let mut other_erc20 = Erc20ContractRef {
```

Comment:
Generated Refs, should have `new(address: Address, env: &Rc<ContractEnv>)`.

Reply:
Ok, I'll fix in the next PR.